src/extensions/registries/page-registry.ts

```diff
@@ -34,21 +34,18 @@ export interface PageRegistration {
    * When not provided, first registered page without "id" would be used for page-menus without target.pageId for same extension
    */
   id?: string;
-  params?: PageParams<string | ExtensionPageParamInit>;
+  params?: PageParams<string | Omit<PageParamInit<any>, "name" | "prefix">>;
```

Comment:
These changes are not related to `mobx6`. Please move them to a different PR.

Reply:
It's partially related because `PageParam` helper depends on `mobx-observable-history` and it was refactored to align with new version.